<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Aleksander Wojdyga</author>
 <class>InstructionWizardDialog</class>
 <widget class="QDialog" name="InstructionWizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create instruction</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabType">
      <attribute name="title">
       <string>Instruction type</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Choose instruction type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignTop</set>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="basicButton">
              <property name="text">
               <string>basic instruction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="selectionButton">
              <property name="text">
               <string>selection instruction (if-then-else)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="iterationButton">
              <property name="text">
               <string>iteration instruction (while-do)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inputButton">
              <property name="text">
               <string>input instruction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="outputButton">
              <property name="text">
               <string>output instruction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="callButton">
              <property name="text">
               <string>procedure call</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="returnButton">
              <property name="text">
               <string>function value return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="clipboardButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>copy from clipboard</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContents">
      <attribute name="title">
       <string>Instruction contents</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QFrame" name="contentsFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>&lt; Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>21</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>InstructionWizardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>InstructionWizardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
